docs/user-guide/configuration.rst

```diff
@@ -171,6 +171,47 @@ Here is a full example showing how to configure all possible sinks:
         api_key: "datadog api key"
         default: false
 
+Configuration secrets
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Some of the configuration values are considered secrets, and cannot be saved in plain text format.
+We recommend using `SealedSecrets <https://github.com/bitnami-labs/sealed-secrets>`_
+or one of the other secret management system for Kubernetes, to encrypt the secret values.
+
+As an alternative, we can pull secret values from Kubernetes secrets.
+
+Here's how:
+
+First, define that the value should be pulled from an environment variable:
+
```

Comment:
I would be more explicit here in the text (and not just the example) that you can use the syntax {{ env.PARAM_NAME }} 

E.g:

  First, define that the value should be pulled from an environment variable by using the special {{ env.VARIABLE }} syntax

Reply:
cool. Fixing